extension/models/Errors/EntityNotFoundError.js

```diff
@@ -0,0 +1,22 @@
+const isString = require('lodash/isString')
+const ERROR_CODE = 'ENOTFOUND'
+
+/**
+ * For truly unknown Magento endpoint errors, e.g. code 500s.
+ * Please consult the magento swagger definition for possible error
+ * outputs and use this error as a fallback to all unknown cases.
+ *
+ * @extends Error
+ * @param {string} [message=An internal error occurred.]
```

Comment:
Signifies default message
```suggestion
 * @param {string} [message=Entity not found.]
```